mid-assessment/q4.rb

```diff
@@ -0,0 +1,66 @@
+# 4. Design regex for mobile number, email address, name, gender(M / F) and amount
+# Personal Info module extract all the details from given text and display it.
+module PersonInfo
+  def self.mobile_number(mystr)
+    puts "Mobile Number : #{mystr.match(/\d{10}/)}"
```

Comment:
Use new line so that the output will be recognized easily.
Example: puts "\nMobile Number: #{mystr.match(/\d{10}/)}"

Reply:
ok